packages/cli/src/workflow-helpers.ts

```diff
@@ -18,41 +16,20 @@ import { VariablesService } from '@/environments.ee/variables/variables.service.
 
 export function generateFailedExecutionFromError(
 	mode: WorkflowExecuteMode,
-	error: NodeApiError | NodeOperationError | WorkflowOperationError,
-	node: INode,
+	error: ExecutionError,
+	node: INode | undefined,
```

Comment:
Do we know in which cases we can have a node-less error?

Reply:
`SubworkflowPolicyChecker.check`